policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax/mi_homestead_property_tax_credit.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class mi_homestead_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan homestead property tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-508",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR.pdf#page=2",
+    )
+    defined_for = "mi_homestead_property_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax.reduction
+        # Line 42
+        total_household_resources = tax_unit("mi_household_resources", period)
+        homestead_allowable_credit = tax_unit(
+            "mi_total_homestead_property_tax_credit", period
+        )
+        # Line 43
+        # The reduction is specified as going from 100% to 0% rather than vice-versa.
+        start = p.start
```

Comment:
use `p.start` directly in the computation for conciseness and to make clear that the value is a parameter (same for others)